README.md

```diff
@@ -298,9 +299,18 @@ my-cluster-zookeeper-nodes    ClusterIP   None             <none>        2181/TC
 
 The service you're interested in is `my-cluster-kafka-bootstrap:9092`. We will use it to deploy the Data Index Service later.
 
-Having the cluster up and running, the next step is creating the `Kafka Topic`s required by the Data Index Service: `kogito-processinstances-events` and `kogito-usertaskinstances-events`.
+#### Create Required Topics
```

Comment:
Is the PR header correct?
shouldn't the topics be created automatically instead?

If my understanding is correct, only the kafka envs are being automatically set.

Reply:
The topics will be created automatically here: https://issues.jboss.org/browse/KOGITO-246